lib/hammer_cli_foreman_virt_who_configure/system_caller.rb

```diff
@@ -9,7 +9,7 @@ def system(commad)
       begin
         @tempfile.write(commad)
         @tempfile.flush # to make sure the command is complete
-        result = Kernel.system("/usr/bin/bash #{@tempfile.path}")
+        result = Kernel.system("env -i HOME=\"$HOME\" LC_CTYPE=\"${LC_ALL:-${LC_CTYPE:-$LANG}}\" PATH=\"$PATH\" USER=\"$USER\" /usr/bin/bash #{@tempfile.path}")
```

Comment:
I'm not sure you need `PATH` since that may pull in the SCL again.

Reply:
I tried to remove the $PATH and command fails, I dont see SCL(/opt/rh) directories in $PATH variable,
~~~
# echo $PATH
/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/opt/puppetlabs/bin:/root/bin
~~~
Can see ruby25 is at path /opt/rh

Reply:
How did you test this? From a production system where Ruby is executed in the SCL?

Reply:
Hello,

I enabled the scl ruby and if I keep the PATH run is successful,

~~~
# scl enable rh-ruby25 bash
# ruby -v
ruby 2.5.5p157 (2019-03-15 revision 67260) [x86_64-linux]

# echo $PATH
/opt/rh/rh-ruby25/root/usr/local/bin:/opt/rh/rh-ruby25/root/usr/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/opt/puppetlabs/bin:/root/bin

# hammer virt-who-config deploy --id 1 --organization-id 1

== [1/5] Installing virt-who ==
Running preparation steps required to run the next scenarios
================================================================================
Check if tooling for package locking is installed:                    [OK]
--------------------------------------------------------------------------------
~~~